docs/sql-ref-syntax-aux-resource-mgmt-add-file.md

```diff
@@ -19,4 +19,32 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`ADD FILE` adds a file, JAR or archive to the list of resources. The added file can be listed using [LIST FILE](sql-ref-syntax-aux-resource-mgmt-list-file.html).
+
+### Syntax
+{% highlight sql %}
+ADD FILE file_name
+{% endhighlight %}
+
+### Parameters
+<dl>
+ <dt><code><em>file_name</em></code></dt>
+ <dd>The name of the file, JAR or archive to be added.</dd>
+</dl>
+
+### Examples
+{% highlight sql %}
+ add FILE /tmp/test1.jar;
```

Comment:
If you add a JAR file as a 'file', it won't be on the classpath right? So I'm not sure if this is a usual usage to show. You'd normally `ADD JAR` right?

Reply:
Right. Adding a JAR file as a 'file' won't be on the classpath. 
I will modify the PR to make it only show ```add JAR``` for JARs and ```add FILE``` for files. Thanks!